sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,16 +1,16 @@
 # Release History
 
-## 5.1.0 (Unreleased)
+## 5.1.0 (2021-07-08)
```

Comment:
not sure if you want to wait until it is definitive as this date can change according to service deployments/other unknowns

Reply:
We can always update later if there is a need to